### ISSUE TITLE
Doesn't work properly with appendChild and removeChild

### ISSUE THREAD

**Issue by akravets12** [rdub80/aframe-gui#12]
I'm dealing with an issue while appending and removing  a-gui-button from a-gui-flex-container. It doesn't append new a-gui-button buttons correctly (they are drawn one on another, not in a row, but before the removal of previous set of a-gui-button  buttons, everything worked just fine. What am I doing wrong?
Before:
![2018-03-27 12 59 54](https://user-images.githubusercontent.com/25844438/37960695-eada7932-31be-11e8-8ba7-01a97db7975d.png)
After:
![2018-03-27 1 00 03](https://user-images.githubusercontent.com/25844438/37960709-f2e895d2-31be-11e8-801a-5759eef3d41a.png)

**Comment by rdub80** [comment]
Do you have any code you can share so I can see what is happening?

**Comment by mauriciomartinez-itesm** [comment]
Hey it's not me, but I'm having the same problem, this is my code:

Javascript
```

var lista = document.querySelector('#lista');
var elemento = document.createElement('a-gui-button');
      elemento.setAttribute('height', '.2');
      elemento.setAttribute('width', '3.5');
      elemento.setAttribute('font-size', '32px');
      elemento.setAttribute('font-color', '#000');
      elemento.setAttribute('background-color', '#FFF');
      elemento.setAttribute('value', st[0]+": "+st[1]);
      lista.appendChild(elemento);
```

HTML
```
<a-gui-flex-container 
        id="lista" flex-direction="column" justify-content="center" align-items="normal" component-padding="0.1" opacity="0.2" scale=".1 .1 .1"
        position="0 0 -.2" rotation="0 0 0">
       </a-gui-flex-container>
```

**Comment by lmalave** [comment]
This is fixed now in the latest release 0.2.1.

Now appending a child triggers a redraw of the container.
